run_beast.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+# This script is intended to invoke beast within a Docker container
+# running on the Google Cloud Platform via the Pipelines API / dsub
+
+IN_DIR=$(dirname "${INPUT_FILE}")
+OUT_DIR=$(dirname "${OUTPUT_FILES}")
+OUTPUT_PREFIX=$(basename "${INPUT_FILE}" .xml)
+
+pwd 
+cd $OUT_DIR
+beast -beagle_info > "${OUTPUT_PREFIX}.out"
+pwd 
+beast -beagle_GPU -beagle_cuda -beagle_double -beagle_scaling always -beagle_order 1 ${INPUT_FILE} >> "${OUTPUT_PREFIX}.out"
```

Comment:
Doesn't the `-beagle_order` option need to scale appropriately with the number of resources/partitions we're given? For example, with 4 partitions & 4 GPUs, you'd want to specify `-beagle_order 1,2,3,4`

Reply:
Yes, this is mostly an example (+for testing). Ideally we'd scale the number of GPUs by `min(num_partitions, requestable_gpu_count_limit)` and expand out `-beagle_order` accordingly. This is an early first pass at all of this...

Reply:
For now I'll parameterize it.